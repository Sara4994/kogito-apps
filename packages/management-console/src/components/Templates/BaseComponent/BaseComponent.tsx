import { BrowserRouter, Route, Switch } from 'react-router-dom';
import PageLayoutComponent from '../PageLayoutComponent/PageLayoutComponent';
import './BaseComponent.css';
import React from 'react';

const BaseComponent: React.FC = () => {
  return (
    <BrowserRouter>
      <Switch>
        <Route path="/" component={PageLayoutComponent} />
      </Switch>
    </BrowserRouter>
  );
};

export default BaseComponent;
