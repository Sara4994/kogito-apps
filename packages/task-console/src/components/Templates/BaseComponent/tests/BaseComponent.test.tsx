import React from 'react';
import { shallow } from 'enzyme';
import BaseComponent from '../BaseComponent';

describe('Base component test cases', () => {
  it('Snapshot testing', () => {
    const wrapper = shallow(<BaseComponent />);
    expect(wrapper).toMatchSnapshot();
  });
});
