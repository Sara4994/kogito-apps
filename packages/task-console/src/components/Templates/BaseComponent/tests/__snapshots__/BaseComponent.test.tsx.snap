// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Base component test cases Snapshot testing 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      component={[Function]}
      path="/"
    />
  </Switch>
</BrowserRouter>
`;
